backend/api/Models/Concepts/Address/AddressMap.cs

```diff
@@ -0,0 +1,36 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Models.Concepts;
+
+namespace Pims.Api.Models.Concepts
```

Comment:
Created a new folder under Models called Concepts (Couldn't think of anything better :) )
THey are isolated from all other models so that we can start refactoring the rest of the controllers. They pair with a models folder on the front end. Moving forward we should only create models there, and remove the other ones as we refactor

Reply:
Maybe we could talk about this during our meeting today, I don't mind the word, but mixing concept with _model seems weird.

Reply:
Not a fan of that naming either. Let's discuss in our meeting. Don't see any problem in them just being "models" and live in the root model folder